### ISSUE TITLE
SMAPI launcher doesn't work for one(?) Linux user

### ISSUE THREAD

**Issue by Pathoschild** [Pathoschild/SMAPI#219]
#183 changed the launcher to open a separate SMAPI terminal on Linux/Mac, matching the behaviour on Windows. This caused an issue where some Linux users couldn't launch SMAPI through Steam, which was mostly fixed by #198 (by launching without a terminal if the terminal returned 127 command-not-found).

The SMAPI terminal launch still fails for [one user on the forums](http://community.playstarbound.com/threads/smapi-stardew-modding-api.108375/page-58#post-3094464), but the reason for that is unclear.

**Comment by Pathoschild** [comment]
I created a [custom launcher](https://github.com/Pathoschild/SMAPI/files/715541/StardewModdingAPI.zip) which logs more information about the launch process.

* ✓ Log for a GOG user (no issue):
  ```
  invoked as ./StardewValley 
  detected Linux (uname=Linux, arch=x86_64)
  selected launcher: ./StardewModdingAPI.bin.x86_64 
  selected command: command -v
  launching terminal: x-terminal-emulator
  terminal launch exit code: 0
  ```
* ✓ Log for a Steam user affected by the command-not-found issue (see #198):
  ```
  invoked as /home/ledah/.local/share/Steam/steamapps/common/Stardew Valley/StardewValley STEAM_RUNTIME=0
  detected Linux (uname=Linux, arch=x86_64)
  selected launcher: ./StardewModdingAPI.bin.x86_64 STEAM_RUNTIME=0
  selected command: command -v
  launching terminal: x-terminal-emulator
  terminal launch exit code: 127
   ```
* ✘ Log from [@nirasa1957 on the forums](http://community.playstarbound.com/threads/smapi-stardew-modding-api.108375/page-58#post-3095165), who can't launch the game through Steam (but it works fine if launched directly):
  ```
   invoked as /home/irenas/.local/share/Steam/steamapps/common/Stardew Valley/StardewValley STEAM_RUNTIME=0
   detected Linux (uname=Linux, arch=x86_64)
   selected launcher: ./StardewModdingAPI.bin.x86_64 STEAM_RUNTIME=0
   selected command: command -v
   launching terminal: x-terminal-emulator
   terminal launch exit code: 1
   ```

**Comment by Pathoschild** [comment]
Apparently fixed with Stardew Valley 1.2 beta and SMAPI 1.9 beta.
